.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -0,0 +1,87 @@
+name: Test windows on the comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  windows_test_preparation:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/windu please' }}
```

Comment:
Could we use some readable words? Like: "Windows please" or "Windows"

Reply:
Done.